From: KevinDuan <duankaiwen@kylinos.com>
Date: Thu, 23 May 2024 09:21:35 +0000
Subject: Translated using Weblate (Uyghur)

Currently translated at 49.5% (60 of 121 strings)

Translation: openkylin-nile-new/kylin-os-manager-driver-controller
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/kylin-os-manager-driver-controller/ug/
---
 .../kylin-os-manager-driver-controller_ug.ts       | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/plugins/driver-controller/translations/kylin-os-manager-driver-controller_ug.ts b/plugins/driver-controller/translations/kylin-os-manager-driver-controller_ug.ts
index 4854f6e..6bf1343 100644
--- a/plugins/driver-controller/translations/kylin-os-manager-driver-controller_ug.ts
+++ b/plugins/driver-controller/translations/kylin-os-manager-driver-controller_ug.ts
@@ -26,12 +26,12 @@
     <message>
         <location filename="../src/deviceitem.cpp" line="56"/>
         <source>Bluetooth</source>
-        <translation>كۆكچىش</translation>
+        <translation>كۆك چىش تەڭشىكى</translation>
     </message>
     <message>
         <location filename="../src/deviceitem.cpp" line="60"/>
         <source>Printer</source>
-        <translation>پرىنتېرلاش</translation>
+        <translation>پرىنتېر</translation>
     </message>
     <message>
         <location filename="../src/deviceitem.cpp" line="64"/>
@@ -212,7 +212,7 @@
     <message>
         <location filename="../src/hardwareinfowidget.cpp" line="218"/>
         <source>Serial</source>
-        <translation>قاتار</translation>
+        <translation>تەرتىپ نومۇرى</translation>
     </message>
     <message>
         <location filename="../src/hardwareinfowidget.cpp" line="366"/>
@@ -248,7 +248,7 @@
     <message>
         <location filename="../src/hardwareinfowidget.cpp" line="517"/>
         <source>Disk</source>
-        <translation>دىسكاز</translation>
+        <translation>قاتتىق دىسكا</translation>
     </message>
     <message>
         <location filename="../src/hardwareinfowidget.cpp" line="543"/>
@@ -273,7 +273,7 @@
     <message>
         <location filename="../src/hardwareinfowidget.cpp" line="649"/>
         <source>Bluetooth</source>
-        <translation>كۆكچىش</translation>
+        <translation>كۆك چىش تەڭشىكى</translation>
     </message>
     <message>
         <location filename="../src/hardwareinfowidget.cpp" line="874"/>
@@ -367,7 +367,7 @@
         <location filename="../src/hardwareinfowidget.cpp" line="1368"/>
         <location filename="../src/hardwareinfowidget.cpp" line="1442"/>
         <source>Type</source>
-        <translation>تۈرى</translation>
+        <translation>كۆرۈنۈش ئۇسلۇبى</translation>
     </message>
     <message>
         <location filename="../src/hardwareinfowidget.cpp" line="1084"/>
@@ -414,7 +414,7 @@
         <location filename="../src/hardwareinfowidget.cpp" line="1219"/>
         <location filename="../src/hardwareinfowidget.cpp" line="1272"/>
         <source>Version</source>
-        <translation>نەشرى</translation>
+        <translation>نەشر نامى</translation>
     </message>
     <message>
         <location filename="../src/hardwareinfowidget.cpp" line="1226"/>
@@ -522,13 +522,13 @@
         <location filename="../src/hardwareinfowidget.cpp" line="1612"/>
         <location filename="../src/hardwareinfowidget.cpp" line="1664"/>
         <source>Yes</source>
-        <translation>شۇنداق</translation>
+        <translation>قوشۇش</translation>
     </message>
     <message>
         <location filename="../src/hardwareinfowidget.cpp" line="1612"/>
         <location filename="../src/hardwareinfowidget.cpp" line="1664"/>
         <source>No</source>
-        <translation>ياق</translation>
+        <translation>ئەمەلدىن قالتۇرۇش</translation>
     </message>
     <message>
         <location filename="../src/hardwareinfowidget.cpp" line="1619"/>
@@ -669,12 +669,12 @@
     <message>
         <location filename="../src/uninstallitem.cpp" line="55"/>
         <source>Bluetooth</source>
-        <translation>كۆكچىش</translation>
+        <translation>كۆك چىش تەڭشىكى</translation>
     </message>
     <message>
         <location filename="../src/uninstallitem.cpp" line="59"/>
         <source>Printer</source>
-        <translation>پرىنتېرلاش</translation>
+        <translation>پرىنتېر</translation>
     </message>
     <message>
         <location filename="../src/uninstallitem.cpp" line="63"/>
